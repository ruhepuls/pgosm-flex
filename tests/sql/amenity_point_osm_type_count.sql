SELECT osm_type COLLATE "C", COUNT(*)
    FROM osm.amenity_point
    GROUP BY osm_type COLLATE "C"
    ORDER BY osm_type COLLATE "C"
;