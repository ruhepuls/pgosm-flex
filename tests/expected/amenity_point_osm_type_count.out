animal_boarding|3
arts_centre|22
atm|41
bank|246
bar|194
bbq|2
bench|563
bicycle_parking|215
bicycle_rental|315
bicycle_repair_station|20
biergarten|3
border_control|1
brewery|1
building|1
bureau_de_change|1
bus_station|9
cafe|334
car_rental|8
car_sharing|43
car_wash|5
catering|8
charging_station|9
check_cashing|5
childcare|3
cinema|8
clinic|29
clock|3
club|2
college|2
community_centre|11
compressed_air|7
convenience|6
courthouse|4
coworking|1
coworking_space|2
crypt|1
dentist|20
doctors|37
dojo|1
drinking_water|173
dry_cleaner|5
embassy|21
events_venue|1
fast_food|439
ferry_terminal|6
fire_station|19
first_aid|1
flower_planter|3
food_court|7
fountain|28
fuel|54
gallery|2
government|1
grave_yard|6
historic:theatre|1
hospital|1
hospital (historic)|5
ice_cream|19
ice_cream;cafe|1
internal_kindergarten|1
internet_cafe|1
kindergarten|161
library|23
marketplace|6
money_transfer|7
motorcycle_parking|3
nail_salon|5
nightclub|16
nursing_home|7
office|1
other|2
parking|122
parking_entrance|40
pharmacy|68
place_of_worship|376
police|25
post_box|177
post_office|56
prep_school|2
pub|97
public_bookcase|14
public_building|11
recycling|33
refuge|1
restaurant|964
school|358
school (historic)|1
shelter|4
social_club|1
social_facility|22
spa|1
stripclub|6
studio|7
tax_service|1
taxi|2
theatre|33
theatre (historic)|1
toilets|29
tourist|22
trade_school|1
tutor|1
university|9
vending_machine|18
veterinary|7
waste_basket|180
waste_disposal|1
